### PR TITLE
More factorio improvements

### PR DESCRIPTION
**Pull request by henriquegemignani**
- Add a few no-science-pack tech
- Shuttle Automation Science Pack
- Redo fitness formula, should work a lot better
- Allow multiple fluids for late science packs

### CODE REVIEW

File: randovania/games/factorio/generator/item_cost.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+def item_is_fluid(item_name: str) -> bool:
+    return item_name in _k_fluids
+
+
+def print_things():
+    recipes_raw = data_parser.load_recipes_raw()
+    techs_raw = data_parser.load_techs_raw()
+    complexities = cost_calculator(recipes_raw, techs_raw)
+
+    from prettytable import PrettyTable
```
Comment (duncathan) [inline randovania/games/factorio/generator/item_cost.py:145]: should this be a dependency? or should this function just be removed?

File: randovania/games/factorio/generator/item_cost.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+import dataclasses
+import math
+
+from randovania.games.factorio.data_importer import data_parser
+
+BASIC_RESOURCES = {"water", "steam", "crude-oil", "iron-ore", "copper-ore", "coal", "stone"}
+_k_fluids = {"water", "steam", "crude-oil", "heavy-oil", "light-oil", "petroleum-gas", "lubricant", "sulfuric-acid"}
```
Comment (duncathan) [inline randovania/games/factorio/generator/item_cost.py:9]: what's the `k` mean? and shouldn't this be allcaps?

**Review by duncathan — CHANGES_REQUESTED** [review]

File: randovania/games/factorio/generator/item_cost.py (hunk 1 above)
Comment (henriquegemignani) [reply randovania/games/factorio/generator/item_cost.py:145]: Probably removed, but it's a very helpful function for debugging this

File: randovania/games/factorio/generator/item_cost.py (hunk 2 above)
Comment (henriquegemignani) [reply randovania/games/factorio/generator/item_cost.py:9]: Constant, which makes it really funny with the other convention for constants one line above

**Review by duncathan — APPROVED** [review]

**Pull request merged**